libs/blocks/src/lib/block-wrapper/index.tsx

```diff
@@ -0,0 +1,42 @@
+import { FluidContainer, Heading, qtMerge, Text } from '@deriv/quill-design';
+import { ReactNode } from 'react';
+
+type bgVariant = 'light' | 'gray';
+
+const bgColorVariantClass: { [key in bgVariant]: string } = {
+  light: 'bg-background-primary-container',
+  gray: 'bg-background-primary-base',
+};
+export interface ContentTextProps {
+  title?: ReactNode;
+  description?: ReactNode;
+  className?: string;
+  children: ReactNode;
+  color: bgVariant;
+}
+
+export const BlockWrapper = ({
+  title,
+  description,
+  className,
+  color,
```

Comment:
Let's change the prop name and add a default value. `color` is usually pertaining to `text color`